content/memory_model.tex

```diff
@@ -54,11 +54,15 @@
 
 \subsection{Atomicity Guarantees}\label{subsec:amo_guarantees}
 
-\openshmem contains a number of routines that operate on symmetric data
-atomically (Section \ref{sec:amo}).  These routines guarantee that accesses by
-\openshmem's atomic operations with the same datatype will be exclusive, but do not guarantee
-exclusivity in combination with other routines, either inside \openshmem or
-outside.
+\openshmem contains a number of routines that perform atomic operations on
+symmetric data objects (Section \ref{sec:amo}).  These routines guarantee that
+concurrent accesses to the same location by \openshmem atomic operations using
+the same datatype will be exclusive.  However, they do not guarantee
+exclusivity when concurrent accesses to the same location are performed using
+\openshmem atomic operations using different datatypes, when non-atomic
```

Comment:
Add wait/test with same datatype

Reply:
@jdinan, you mean to add wait/test with same datatype to the allowed/defined list, correct?